packages/terra-table/README.md

```diff
@@ -1,10 +1,9 @@
 # Terra Table
 
-
 [![NPM version](https://badgen.net/npm/v/terra-table)](https://www.npmjs.org/package/terra-table)
 [![Build Status](https://badgen.net/travis/cerner/terra-core)](https://travis-ci.com/cerner/terra-core)
 
-The Terra Table is a structural component to arrange content within table.
+The terra-table component provides users a way to render data in a tabular format.
```

Comment:
We should remove `Internationalization Support` and `Localization Support` from the list of component feature since there are no translations

Reply:
Updated.